src/intTest/resources/edifact/multi_transaction.1.dat

```diff
@@ -0,0 +1,55 @@
+UNB+UNOA:2+TES5+XX11+920116:1529+00000007'
+UNH+00000008+FHSREG:0:1:FH:FHS001'
+BGM+++507'
+NAD+FHS+XX1:954'
+DTM+137:199201171259:203'
+RFF+950:G2'
```

Comment:
Now its a G2 (outbound amendment) transaction which isn't quite right either. I'll try to find some time to do a multi message / transaction approval/rejection, but you might be able to build one by coming a few appendix j examples.